src/monitors/Moderation/newLineFilter.ts

```diff
@@ -45,7 +45,7 @@ export default class extends ModerationMonitor {
 
 	protected onLogMessage(message: GuildMessage, language: Language) {
 		return new MessageEmbed()
-			.splitFields(message.content)
+			.setDescription(message.content)
```

Comment:
You're sure about these and the others? I honestly never fully understood `splitFields` but just checking.

Reply:
Sets fields when they don't fit into the description.

message.content (2000 characters maximum) always fits in embed.description (2048 characters maximum).